R/GetReport.R

```diff
@@ -26,13 +27,14 @@
 #' @export
 #'
 
-GetReport <- function(report.id,interval.seconds=10,max.attempts=3,print.attempts=TRUE) {
+GetReport <- function(report.id,interval.seconds=10,max.attempts=3,print.attempts=TRUE,format="json") {
 
   request.body <- c()
   request.body$reportID <- unbox(report.id)
-  report.data <- ApiRequest(body=toJSON(request.body),func.name="Report.Get",interval.seconds=interval.seconds,max.attempts=max.attempts,print.attempts=print.attempts)
+  request.body$format <- unbox(format)
+  report.data <- ApiRequest(body=toJSON(request.body),func.name="Report.Get",interval.seconds=interval.seconds,max.attempts=max.attempts,print.attempts=print.attempts,format=format)
 
-  report.type <- report.data$report$type
+  if(format!="csv") report.type <- report.data$report$type else report.type <- NULL
```

Comment:
As a matter of style, can you take this off of one-line and use curly-braces, just to make it more obvious that it is a logic branch that sets `report.type`?